pkg/extension/opampextension/README.md

```diff
@@ -68,7 +67,7 @@ option:
 
 Here is a list of valid values for the OpAMP `endpoint** configuration option:
 
-**Note:** As of Jan 2023, these endpoints are not yet available.
```

Comment:
Do we need to specify date? It requires update once per month and as we can easily miss it

Reply:
Good point, removing date, it's unnecessary.